doc/src/Thumbnails.dox

```diff
@@ -30,7 +30,7 @@
  * If supported by the host, it will then:
  * - Call @ref preflight with the target entity and a @needsref ThumbnailSpecification
  *   that describes supported formats.
- * - Call @ref openassetio.managerAPI.ManagerInterface.ManagerInterface.getEntityMetadata
+ * - Call @ref openassetio.managerAPI.ManagerInterface.ManagerInterface.getEntityAttributes
  *   "getEntityMetadata" on the returned reference to determine the desired width/height/format
```

Comment:
I'm a bit confused by this. I see `ManagerInterface` has a `getEntityAttributes` and not a `getEntityMetadata` method, what's going on with the `"getEntityMetadata"` appearing on the next line?

Btw, searching the codebase shows up `getEntityMetadata` in a few other places - should they all be fixed up?